.github/TriageProcess.md

```diff
@@ -0,0 +1,97 @@
+# Triage Process
+
+Managing a popular GitHub repo with a small team is not an easy task. It requires a good balance between creating new features while handling many investigations and bug fixes associated with existing ones.
+
+During our time working on Xamarin.Forms and while ramping up with .NET MAUI, the amount of incoming issues has been constantly growing. While this is a sign of a healthy framework and ecosystem surrounding it, it's becoming harder to react to all those issues.
+To be able to keep up with ever-evolving expectations, we're introducing a set of rules to help us better handle the incoming issues going forward.
+
+**Note:** Customers that need help with **urgent investigations** should contact [Microsoft Support](https://dotnet.microsoft.com/platform/support).
+
+## Goals
+
+The goals of these rules are listed below in priority order:
+
+- Make it easy to make triage decisions for each issue filed in this repository
+- Be able to easily prioritize issues for each milestone
+- Set proper expectations with customers regarding how issues are going to be handled
+
+## Triage Process Details
+
+The feature teams should be able look through every single issue filed against this repository and be able to make a quick call regarding the nature of the issue.
+We will first categorize the issues and further handle these depending on the category the issue is in. The subsections below represent these categories and the rules we apply for them during our triage meeting.
+
+### Information Gathering
+
+In this phase we instruct the user on how to collect the appropriate diagnostics and see if they are able to address the issue with that additional information.  When we need user input we will mark the issue with `s/needs-info` label. Issues in this phase may be closed automatically if we do not receive timely responses, they often do not provide enough information for us to investigate further.
+We'll try to respond quickly to such issues (within days). If a user has collected all of the relevant diagnostics and the issue is still not apparent, then we will consider it for further [investigation](#investigations) by the team.
+
+### Feature requests
+
+As soon as we identify an issue represents an ask for a new feature, we will label the issue with the `t/enhancement ☀️` label.
```

Comment:
Hooks into the comment on the bug report template: Should we not add any labels by default at all?

Reply:
Similar to the "bug" label, I think if we have specific templates for bug, feature request, etc. then we might as well apply the label in the template. We can always change the labels if they're incorrect.